README.md

```diff
@@ -97,12 +109,27 @@ If you've installed the package from source using poetry, you can still use the
 poetry run python openhands_resolver/resolve_issues.py --repo all-hands-ai/openhands-resolver
 ```
 
+
+
+
+## Responding to PR Comments
+
+The resolver can also respond to comments on pull requests. When a comment is made on a PR:
+
+1. The resolver analyzes the comment and the PR context
+2. Understands the requested changes or questions
+3. Makes appropriate code modifications if needed
+4. Responds to the comment with explanations or updates
+5. Can update the PR with new commits if code changes are required
+
+This functionality is available both through the GitHub Actions workflow and when running the resolver locally.
```

Comment:
Please provide the actual command that can be run here, similarly to how the command is provided in the "## Resolving Issues" section.